tools/templates/nova_lfe/vm.args

```diff
@@ -0,0 +1,6 @@
+-sname '{{name}}'
+
+-setcookie '{{name}}_cookie'
+
++K true
++A30
```

Comment:
This file could also be reused from our existing file I guess?

Reply:
we don't have any vm.args file in nova directory that I could see.

Reply:
Oh. Then it might be a good thing to add it :). Could you do it in your commit? I think we can use the same file for both of them .